docs/src/simulation_tips.md

```diff
@@ -215,29 +215,28 @@ For example if can be difficult to just view a `CuArray` since Julia needs to ac
 its elements to do that. Consider the example below:
 
 ```julia
```

Comment:
Should we turn the examples here into doctests?

Reply:
Don't they need GPU?
Our docs are built on CPU...